freqtrade/freqai/prediction_models/BaseClassifierModel.py

```diff
@@ -68,7 +68,7 @@ def train(
         return model
 
     def predict(
-        self, unfiltered_dataframe: DataFrame, dk: FreqaiDataKitchen, first: bool = False
+        self, dataframe: DataFrame, dk: FreqaiDataKitchen, first: bool = False, **kwargs
     ) -> Tuple[DataFrame, npt.NDArray[np.int_]]:
```

Comment:
Any reason you changed this name?

I actually agree it is too long. I would prefer, however, `unfiltered_df` if possible?

Reply:
I changed it to fix the mypy errors. Renaming it to unfiltered_df causes this error:

`freqtrade/freqai/prediction_models/BaseClassifierModel.py:70: error: Signature of "predict" incompatible with supertype "IFreqaiModel"
freqtrade/freqai/prediction_models/BaseClassifierModel.py:70: note:      Superclass:
freqtrade/freqai/prediction_models/BaseClassifierModel.py:70: note:          def predict(self, dataframe: Any, dk: FreqaiDataKitchen, first: bool = ..., **kwargs: Any) -> Tuple[Any, ndarray[Any, dtype[signedinteger[Any]]]]
freqtrade/freqai/prediction_models/BaseClassifierModel.py:70: note:      Subclass:
freqtrade/freqai/prediction_models/BaseClassifierModel.py:70: note:          def predict(self, unfiltered_df: Any, dk: FreqaiDataKitchen, first: bool = ..., **kwargs: Any) -> Tuple[Any, ndarray[Any, dtype[signedinteger[Any]]]]
Found 1 error in 1 file (checked 195 source files)`

Reply:
well that suggests that the parent class is using `dataframe` (which it does).
so it's surprising that it's not failing right now on develop - but to fix it, all that's really needed is a change in the signature in freqai_interface.

Reply:
Didn't want to change interface, it can be done of course.

Reply:
I was unaware that the argument names needed to be identical - I though position was sufficient. 

It is ok, I fixed it.